### ISSUE TITLE
Unable to open any savec fits post Version 2.1.1

### ISSUE THREAD

**Issue by Luke-Antra** [pyfa-org/Pyfa#1719]

**Comment by blitzmann** [comment]
Looks like there was a bad database migration. Will check more in depth when I get home. In the meantime, can you send me your  database to sable.blitzmann@gmail.com. it should be under `%userprofile%/.pyfa/saveddata.db`

**Comment by Luke-Antra** [comment]
Database sent as requested.

**Comment by blitzmann** [comment]
Thanks @Luke-Antra!

So, yeah, seems like it was a database migration issue. Not exactly sure how or why it would happen, but your user database is set to version 28 without it having version 28 schema (a couple of columns were added to the modules table).

This might happen on edge cases, for example, trying a dev version that doesn't have a completely fleshed out migration script yet, or maybe even downgrading at some point. But not sure what went wrong here.

Can you also send me all the log files in that directory? Hopefully I can find where the version was bumped up and see if there were any errors associated with it.

In the meantime, I will fix this database and send it back to you so that you can use it with newer pyfa versions. :)

**Comment by Luke-Antra** [comment]
I've attached all logfiles contained in `%userprofile%/.pyfa/` 
The fixed database works, thanks a lot for your help! :)

I also have all the saveddata_migration_<date>.db files if those are of any help.

[Pyfa-2018-05-07.log](https://github.com/pyfa-org/Pyfa/files/2324659/Pyfa-2018-05-07.log)
[Pyfa-2018-05-08.log](https://github.com/pyfa-org/Pyfa/files/2324660/Pyfa-2018-05-08.log)
[pyfa-2018-08-19.log](https://github.com/pyfa-org/Pyfa/files/2324661/pyfa-2018-08-19.log)
[pyfa-2018-08-20.log](https://github.com/pyfa-org/Pyfa/files/2324662/pyfa-2018-08-20.log)
[pyfa-2018-08-21.log](https://github.com/pyfa-org/Pyfa/files/2324663/pyfa-2018-08-21.log)
[pyfa-2018-08-23.log](https://github.com/pyfa-org/Pyfa/files/2324664/pyfa-2018-08-23.log)
[pyfa-2018-08-26.log](https://github.com/pyfa-org/Pyfa/files/2324665/pyfa-2018-08-26.log)
[Pyfa_debug-2018-01-22.log](https://github.com/pyfa-org/Pyfa/files/2324666/Pyfa_debug-2018-01-22.log)
[Pyfa-2018-05-06.log](https://github.com/pyfa-org/Pyfa/files/2324667/Pyfa-2018-05-06.log)

**Comment by blitzmann** [comment]
> I also have all the saveddata_migration_.db files if those are of any help.

Yeah, actually, if you could zip those up and email them to me for further analysis.

Will sort through the log files. I'm not really expecting much, but want to make sure :)

**Comment by blitzmann** [comment]
Got the other database files, thanks!

Will try to wade through all the datas, but I suspect it was an edge case that should hopefully be resolved now. :)
